### PR TITLE
Symfony 3.3

### PR DESCRIPTION
**Pull request by j0k3r**
| Q             | A
| ------------- | ---
| Bug fix?      | maybe?
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #...
| License       | MIT

Update to Symfony 3.3 and update restriction deps for all other deps (but some aren't compatible ATM, like this one https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1768).

For an easy review, you can avoid the `CS` commit and only check the other one.

### CODE REVIEW

**Comment by j0k3r** [comment]
Waiting for https://github.com/K-Phoen/rulerz-bridge/pull/1 to be merged by @K-Phoen :clock1030:

**Comment by tcitworld** [comment]
@K-Phoen

**Comment by tcitworld** [comment]
Rulerz-bridge 1.0.1 is out.

File: app/config/config.yml [hunk 1]
```diff
@@ -350,3 +350,10 @@ fos_js_routing:
         - howto
         - fos_user_security_logout
         - new
+
+jms_serializer:
+    handlers:
+        # to be removed if we switch to (default) ISO8601 datetime instead of ATOM
+        # see: https://github.com/schmittjoh/JMSSerializerBundle/pull/494
+        datetime:
+            default_format: "Y-m-d\\TH:i:sO" # ATOM
```
[the remark this answers is not in the document]
Comment (j0k3r) [reply app/config/config.yml]: Yep but should use the default one or keep the old one?

**Pull request merged**